### PR TITLE
Added frequently used emojis support

### PR DESCRIPTION
**Pull request by mananjadhav**
### Details

- Added support for Frequently Used Emojis and Recently used emojis
- Belongs to a common list and it is synced across sessions with NameValuePairs

### Fixed Issues

$ https://github.com/Expensify/App/issues/4559

### Tests
1. Tested adding multiple emojis to recent
2. Tested emojis by multiple counts (frequency)
3. Tested by limiting the emoji count and relevant replacement logic in case of more emojis than the given limit
4. Tested sync across sessions for Frequent Emoji List
5. Tested preferred skin tone along with frequent emojis

### QA Steps
1. Go to any chat and open EmojPicker
2. Select your favorite emoji and add it to Text
3. Open the EmojiPicker again and see it as a part of the Frequently Used Emojis
4. Add multiple emojis to see that the order is either based on the frequency or recently added emojis

### Tested On

- [x] Web
- [x] Mobile Web
- [x] Desktop
- [x] iOS
- [x] Android

#### Web

web-frequent-emojis

#### Mobile Web

mweb-frequent-emojis

#### Desktop

desktop-frequent-emojis

#### iOS

ios-frequent-emojis

#### Android

android-frequent-emojis

### CODE REVIEW

**Comment by mananjadhav** [comment]
@stitesExpensify While my work is done, I've got a regression from the existing `main` branch, where EmojiPicker has some extra padding on the left. 

image

**Comment by mananjadhav** [comment]
@stitesExpensify I've fixed the above padding issue with a small fix in `Popover/index.native.js`. Please review that as well.

**Comment by Jag96** [comment]
@mananjadhav https://github.com/Expensify/App/pull/6523 has fixed the padding issue so if [this commit](https://github.com/Expensify/App/pull/6498/commits/c27d334f38448ed50ec1a3f65171d44ff6ab08a5) is no longer necessary please remove it

**Comment by mananjadhav** [comment]
@Jag96 yeah it won't be needed I'll take the latest pull of main and remove my commit.

**Comment by mananjadhav** [comment]
Commit removed and PR updated. @stitesExpensify Any updates for me on this one?

**Comment by Jag96** [comment]
@stitesExpensify is OOO, assigning pullerbear to get another reviewer on this one

**Review by Jag96 — COMMENTED** [review]
Works well! A few requests, will let @stitesExpensify do a final review when he's back from OOO

**Comment by mananjadhav** [comment]
@Jag96 PR updated

File: src/libs/actions/User.js
Comment (Jag96) [inline src/libs/actions/User.js:326]: Missed an `{Array}` usage here

**Review by Jag96 — CHANGES_REQUESTED** [review]
Just one missed usage, otherwise LGTM

File: src/libs/actions/User.js
Comment (mananjadhav) [reply src/libs/actions/User.js]: Sorry about that. Fixed and updated PR.

**Review by Jag96 — APPROVED** [review]
Thanks @mananjadhav, this LGTM, @stitesExpensify when you get back from OOO feel free to leave a review if there are any follow-ups you'd like addressed.

**Comment by stitesExpensify** [comment]
:fire: I didn't review the code but I just tested it and it seems to work great! Thanks for the work @mananjadhav and thanks for taking over the review for me @Jag96 I appreciate it

**Pull request merged**
